gc/base/EnvironmentBase.cpp

```diff
@@ -151,7 +156,7 @@ MM_EnvironmentBase::tearDown(MM_GCExtensionsBase *extensions)
 #endif /* OMR_GC_SEGREGATED_HEAP */
 
 #if defined(OMR_GC_MODRON_SCAVENGER) || defined(OMR_GC_VLHGC)
-	_hotFieldStats.tearDown(this);
+	_hotFieldStats->kill(this->getExtensions());
```

Comment:
missed check for (NULL != _hotFieldStats)

Reply:
👍